deploy/ui/form.json

```diff
@@ -1597,7 +1597,7 @@
                       "label": "CPU request",
                       "type": "text",
                       "jsonPath": "$.spec.objects.smartRouter.resources.requests.cpu",
-                      "default": "500m",
+                      "default": "1000m",
```

Comment:
smartrouter does not need such value, maybe 250m for minimum and 500 for maximum.